src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -518,6 +518,28 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 			with: cellModel,
 			onPrimaryAction: { [weak self] in
 				self?.viewModel.didTapTestResultButton(coronaTestType: coronaTestType)
+			},
+			onSecondaryAction: { [weak self] in
+				let alert = UIAlertController(
+					title: AppStrings.Home.TestResult.ShownPositive.deleteAlertTitle,
+					message: AppStrings.Home.TestResult.ShownPositive.deleteAlertDescription,
+					preferredStyle: .alert
+				)
+				
+				let deleteAction = UIAlertAction(
+					title: AppStrings.AntigenProfile.Profile.deleteAlertDeleteButtonTitle,
+					style: .destructive,
+					handler: { [weak self] _ in
+						self?.viewModel.coronaTestService.removeTest(coronaTestType)
+					}
+				)
+				deleteAction.accessibilityIdentifier = AccessibilityIdentifiers.Home.ShownPositiveTestResultCell.deleteButton
+				alert.addAction(deleteAction)
+				
+				let cancelAction = UIAlertAction(title: AppStrings.Common.alertActionCancel, style: .cancel)
+				alert.addAction(cancelAction)
+				
+				self?.present(alert, animated: true, completion: nil)
```

Comment:
Not blocking, but I think it would be nicer to have the displaying of an Alert handled by a coordinator and not inside a "dumb" view controller.

Reply:
I totally agree that it coordinator is a place where it should be however, there are at least three more places where we are handling it in view controller, so will to keep it here for now for consistency purposes. Probably, we could refactor it as a utility later on.

Reply:
create a ticket so we do not forget that.